src/pages/docs/api/control-api.tsx

```diff
@@ -1,16 +1,21 @@
+import { useLocation } from '@reach/router';
+import { Link, withPrefix } from 'gatsby';
+import Icon from '@ably/ui/core/Icon';
 import { useSiteMetadata } from '../../../hooks/use-site-metadata';
 import { Head } from '../../../components/Head';
 import { Loader } from '../../../components/Redoc';
-import { Link, withPrefix } from 'gatsby';
-import Icon from '@ably/ui/core/Icon';
 
 const ControlApi = () => {
+  const location = useLocation();
   const { canonicalUrl } = useSiteMetadata();
   const canonical = canonicalUrl('/docs/api/control-api');
   const meta_title = 'Control API';
   const meta_description =
     'The Control API is a REST API that enables you to manage your Ably account programmatically. This is the Control API Reference guide.';
-  const controlAPI = withPrefix('/open-specs/control-v1.yaml');
+  const controlAPI =
+    process.env.NODE_ENV === 'production' && location.hostname === 'ably.com'
+      ? 'https://docs.ably.com/open-specs/control-v1.yaml'
+      : withPrefix('/open-specs/control-v1.yaml');
```

Comment:
There is also the undocumented [`withAssetPrefix`](https://github.com/gatsbyjs/gatsby/blob/346314d678d9465c998c12466f810b21592f6cb5/packages/gatsby-link/src/index.js#L11-L13) that _might_ simplify this 🤞 

~~~typescript
import { withAssetPrefix } from 'gatsby';
const controlAPI = withAssetPrefix('/open-specs/control-v1.yaml');
~~~